interactions/api/models/guild.py

```diff
@@ -2019,6 +2020,157 @@ async def delete_emoji(
+        :rtype: Sticker
+        """
+        if not self._client:
+            raise LibraryException(code=13)
+
+        _name = name if name is not MISSING else file._filename
+
+        payload: dict = {
+            "name": _name,
+            "tags": tags,
+        }
+        if description is not MISSING:
+            payload["description"] = description
+
+        res = await self._client.create_guild_sticker(
+            payload=payload, file=file, guild_id=int(self.id), reason=reason
+        )
+
+        if self.stickers is None:
+            self.stickers = []
+
+        _sticker = Sticker(**res)
+        self.stickers.append(_sticker)
+        return _sticker
+
+    async def modify_sticker(
+        self,
+        sticker_id: Union[Sticker, Snowflake, int],
+        name: Optional[str] = MISSING,
+        description: Optional[str] = MISSING,
+        reason: Optional[str] = None,
+    ) -> Sticker:
+        """
+        Modifies the sticker of the guild.
+
+        :param sticker_id: The sticker or ID of the sticker.
+        :type sticker_id: Union[Sticker, Snowflake, int]
+        :param name: The name of the sticker.
+        :type name: Optional[str]
+        :param description: The description of the sticker.
+        :type description: Optional[str]
+        :param reason: The reason of the modification.
+        :type reason: Optional[str]
+        :return: Modified sticker.
+        :rtype: Sticker
+        """
+        if not self._client:
+            raise LibraryException(code=13)
+
+        _id = (
+            sticker_id.id
+            if isinstance(sticker_id, Sticker)
+            else int(sticker_id)
+            if isinstance(sticker_id, Snowflake)
+            else sticker_id
+        )
+
+        payload: dict = {}
+
+        if name is not MISSING:
+            payload["name"] = name
+        if description is not MISSING:
+            payload["description"] = description
+
+        res = await self._client.modify_guild_sticker(
+            payload=payload, guild_id=int(self.id), sticker_id=_id, reason=reason
+        )
+        _sticker = Sticker(**res)
+        for sticker in self.stickers:
+            if sticker.id == _sticker.id:
+                sticker.update(res)
+                return sticker
+
+    async def delete_sticker(
+        self, sticker_id: Union[Sticker, Snowflake, int], reason: Optional[str] = None
+    ):
+        """Deletes the sticker of the guild.
+
+        :param sticker_id: The sticker or ID of the sticker.
+        :type sticker_id: Union[Sticker, Snowflake, int]
+        :param reason: The reason of the deletion.
+        :type reason: Optional[str]
+        """
+        if not self._client:
+            raise LibraryException(code=13)
+
+        _id = (
+            sticker_id.id
+            if isinstance(sticker_id, Sticker)
+            else int(sticker_id)
+            if isinstance(sticker_id, Snowflake)
+            else sticker_id
+        )
```

Comment:
```suggestion
        _id = (
            int(sticker_id.id)
            if isinstance(sticker_id, Sticker)
            else int(sticker_id)
        )
```
If that fails the user fucked it up, no need to check for snowflake